app/src/test/java/comcmput301f16t01/github/carrier/AcceptingTests.java

```diff
@@ -0,0 +1,91 @@
+package comcmput301f16t01.github.carrier;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+/**
+ * Created by michael on 11/10/16.
+ */
+
+public class AcceptingTests {
+
+    /**
+     * (US 05.01.01) As a driver, I want to accept a request I agree with and accept that offered payment upon completion.
```

Comment:
The user case specifies "accept that offered payment upon completion". This is not addressed anywhere in the test.